### PR TITLE
oss-fuzz adjustments and other cleanups

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/fuzz/fuzz-unit-file.c [hunk 1]
```diff
@@ -48,7 +57,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 
                 ll = l + strspn(l, WHITESPACE);
 
-                if (startswith(ll, "ListenNetlink=")) {
+                if (MSAN && startswith(ll, "ListenNetlink=")) {
```
Comment (evverx) [inline src/fuzz/fuzz-unit-file.c]: Would it make sense to put a similar check in `cg_slice_to_path`, where [`FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION`](https://github.com/systemd/systemd/blob/1238250f1dca07b8fcef4d7ebe86891334819e9e/src/basic/cgroup-util.c#L1979) is currently used to detect whether the code is run under `msan`?

Comment (keszybz) [reply src/fuzz/fuzz-unit-file.c]: Done.

**Comment by evverx** [comment]
It seems that the CI has started to fail because some commits have disappeared. I think the tests should pass again after the missing commits is back. @keszybz could you push the branch again?

**Comment by keszybz** [comment]
Oops, sorry, I rebased and pushed the wrong branch.

**Comment by poettering** [comment]
lgtm

**Comment by evverx** [comment]
The CI seems to have been stuck since yesterday. I'm going to run the tests locally and merge the PR afterwards, but it would be great to somehow unblock the CI. @martinpitt, @xnox is there any chance you could take a look at what is happening to the CI?

**Comment by martinpitt** [comment]
@evverx: Sorry, I can't, I don't have access to that infrastructure any more. @xnox or @iainlane have.

**Comment by iainlane** [comment]
On Tue, Mar 27, 2018 at 11:32:56PM -0700, Martin Pitt wrote:
> @evverx: Sorry, I can't, I don't have access to that infrastructure any more. @xnox or @iainlane have.

The job that updates the status on PRs was crashing, because there were
0-length files and it wasn't equipped to ignore those properly.

This PR was one of those 0 length files. We don't have any logging to
say how that happened, I'm afraid. I'll keep an eye and beef it up if
this happens more often. In the meantime I made the job send mail if it
goes wrong instead of dumping errors to a file hidden in the back room
of a machine buried in my cellar.

-- 
Iain Lane                                  [ iain@orangesquash.org.uk ]
Debian Developer                                   [ laney@debian.org ]
Ubuntu Developer                                   [ laney@ubuntu.com ]

**Comment by evverx** [comment]
@iainlane thank you for making the tests green.

@martinpitt thank you for pointing me in the right direction. I was also wondering if https://www.freedesktop.org/wiki/Software/systemd/autopkgtest/ should be updated so that it would be a little bit easier to find a way to report infrastructure issues without disturbing you.

**Comment by martinpitt** [comment]
@evverx: No worries; I updated the wiki page accordingly. Nice to see the tests run again!

@iainlane: Thanks!

**Pull request merged**